src/framework/uicomponents/view/menuitem.h

```diff
@@ -74,6 +74,8 @@ class MenuItem : public QObject, public async::Asyncable
 
     Q_PROPERTY(QList<MenuItem*> subitems READ subitems NOTIFY subitemsChanged)
 
+    Q_PROPERTY(QString nativeMenuBarIconPath READ nativeMenuBarIconPath NOTIFY actionChanged)
```

Comment:
Looks like you're not using nativeMenuBarIconPath

Reply:
It seems the changes to `PlatformMenuBar.qml` got lost